cfd/combustion/gnns/ci/generate_synthetic_data.py

```diff
@@ -16,11 +16,12 @@
 from yaml import dump
 from numpy import zeros
 import sys
-import config
 import os
 
 sys.path.insert(1, "/".join(os.path.realpath(__file__).split("/")[0:-2]))
 
+import config  # noqa:
```

Comment:
```suggestion
import config  # noqa: <the lint error>
```

Could you please add the actual lint error corresponding to the line (to be done for each `` # noqa: `` usage)